task/backend/meta.proto

```diff
@@ -25,7 +25,7 @@ message StoreTaskMeta {
   string effective_cron = 5;
 
   // Task's configured delay, in seconds.
-  int32 offset = 6;
+  string offset = 6;
```

Comment:
I guess this will need to be called out as a breaking change. I'm not sure what will fail when reading back an existing record that was int32 here. I think it's fine to break in this way rather than worrying about backwards compatibility for now.